src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/EligibilityFailureReason.java

```diff
@@ -1,5 +1,6 @@
 package uk.gov.companieshouse.confirmationstatementapi.eligibility;
 
 public enum EligibilityFailureReason {
-    INVALID_COMPANY_STATUS
+    INVALID_COMPANY_STATUS,
+    INVALID_COMPANY_TYPE
```

Comment:
It'd be better if for the type we are more specific as it would help the web journey as well.
Something like INVALID_COMPANY_TYPE_NO_CS01_REQUIRED